bin/cli.ts

```diff
@@ -81,7 +85,7 @@ program
         "--url-is-view",
         "the url is the view url, don't try to find the correct view",
     )
-    .option("-q --quiet", "be quiet")
+    .option("-q --quiet", "be quiet and don't stream out the members")
```

Comment:
on stdout right?

Reply:
Yes, this is for the CLI only. The client library will always emit the member events

Reply:
Clarified in the CLI description here https://github.com/rdf-connect/ldes-client/pull/94/commits/3035c0a04609093daf52711c7cac3fa00dfbedda